lib/discourse_perspective.rb

```diff
@@ -73,10 +73,10 @@ def self.extract_value_from_analyze_comment_response(response)
 
   def self.flag_on_scores(score, post)
     if score[:score] > SiteSetting.perspective_flag_post_min_toxicity
-      PostAction.act(
+      PostActionCreator.create(
         Discourse.system_user,
         post,
-        PostActionType.types[:notify_moderators],
+        :notify_moderators,
```

Comment:
Accepts a symbol as third argument, based on examples in core:

https://github.com/discourse/discourse/blob/main/app/services/spam_rule/flag_sockpuppets.rb#L48